go.sum

```diff
@@ -247,8 +247,8 @@ github.com/containernetworking/plugins v0.9.1/go.mod h1:xP/idU2ldlzN6m4p5LmGiwRD
 github.com/containers/buildah v1.23.0 h1:qGIeSNOczUHzvnaaOS29HSMiYAjw6JgIXYksAyvqnLs=
 github.com/containers/buildah v1.23.0/go.mod h1:K0iMKgy/MffkkgELBXhSXwTy2HTT6hM0X8qruDR1FwU=
 github.com/containers/common v0.44.0/go.mod h1:7sdP4vmI5Bm6FPFxb3lvAh1Iktb6tiO1MzjUzhxdoGo=
-github.com/containers/common v0.44.1 h1:B4cSH5ZJS69Kkum0hFwvX3TL9XSmEXj5b+f5okFzAVg=
-github.com/containers/common v0.44.1/go.mod h1:7sdP4vmI5Bm6FPFxb3lvAh1Iktb6tiO1MzjUzhxdoGo=
+github.com/containers/common v0.46.0 h1:95zB7kYBQJW+aK5xxZnaobCwoPyYOf85Y0yUx0E5aRg=
```

Comment:
This may be a problem - I think we've already vendor-danced and settled on 0.44 - @vrothberg @TomSweeneyRedHat Can you folks confirm?

Reply:
Good catch, @mheon!

I don't think we did the vendor dance yet.  If we did, then we _must_ stick v0.44.X. Otherwise, we still have time.

Reply:
@TomSweeneyRedHat are you human, or are you dancer?

Reply:
We want the new c/common for the new machine fields, otherwise we would have a breaking change in the next version because the `Engine.MachineImage` field no longer exists.

Reply:
Also see https://github.com/containers/podman/issues/11745, arch users already have the new containers.conf

Reply:
@ashley-cui if you were to backport https://github.com/containers/common/pull/782 into the c/common v0.44 branch on c/common and then we created a v.0.44.2 release, would that work for you or do you have other dependencies?

Reply:
lets talk about this one today ... i'm not so sure im comfy with either option.